lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -0,0 +1,267 @@
+import 'dart:ui';
+
+import 'package:equatable/equatable.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:fl_chart/src/utils/lerp.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+typedef GetTitleByIndexFunction = String Function(int index);
+
+class RadarChartData extends BaseChartData {
+  //ToDo(payam) : document data classes
+  RadarChartData({
+    this.dataSets = const [],
+    this.fillColor = Colors.transparent,
+    this.chartBorderData = const BorderSide(color: Colors.black, width: 2),
+    this.radarTouchData,
+    this.getTitle,
+    @required this.titleCount,
+    this.titleTextStyle = const TextStyle(color: Colors.black, fontSize: 12),
+    this.titlePositionPercentageOffset = 0.2,
+    @required this.tickCount,
```

Comment:
It is something like `maxY` in our BarChart. we didn't force to enter it, because we can calculate it using the dataset.
I mean, we will calculate it if it is null.

I think it is better to do that here too.

Reply:
I think we can do the same for `titleCount`.

BTW it is nice to have, and I don't force you to do that.

Just check and do it if you have time. Otherwise, we can merge it and improve it gradually.

Thanks!

Reply:
Yes, you are right. we can calculate it with Datasets. So I remove it from class fields and convert it to a getter.